include/openssl/ssl.h

```diff
@@ -1899,6 +1899,15 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client);
 # ifndef OPENSSL_NO_CT
 
 /*
+ * A callback for verifying that the received SCTs are sufficient.
+ * Expected to return 1 if they are sufficient, otherwise 0.
+ * May return a negative integer if an error occurs.
```

Comment:
@RJPercival why does the callback need negative returns?

As for "will be aborted" clarify who's responsible for aborting it and how/when?